src/modals/CenteredModal.tsx

```diff
@@ -191,7 +191,7 @@ const BackButton = styled(Button)`
 export const ModalContent = styled.div`
   display: flex;
   flex-direction: column;
-  padding: var(--spacing-4);
+  padding: var(--spacing-4) var(--spacing-4) 0 var(--spacing-4);
```

Comment:
Why this change? Now the Action buttons are really close to the modal bottom and it's less balanced IMO. What I tried to achieve before:
- Padding around action button when they are not floating above content
- Reduced padding when they are, to give more space to content.

Reply:
Now that I'm talking padding : we could increase lateral padding to be closer to the mockups and give more space to the scrollbar. Here's what it looks like with `var(--spacing-4) var(--spacing-6)` :

<img width="640" alt="image" src="https://user-images.githubusercontent.com/6715943/235579739-a3ba82ae-451e-4e25-a45f-43b93c9654c1.png">


What do you think?

Reply:
I mentioned the reason in the commit and I tagged you!

Reply:
LGTM 👍